spec/jobs/request_job_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe RequestJob, type: :job do
```

Comment:
These tests are useless. They just confirm that `ApplicationJob#perform_later` works correctly. Try emptying the body of `perform` in RequestJob and your tests won't catch it.

Reply:
Ok. I will try to add some HTTP request mocks or perform real requests as the API I am using is intended for tests.

Reply:
> These tests are useless. They just confirm that `ApplicationJob#perform_later` works correctly. Try emptying the body of `perform` in RequestJob and your tests won't catch it.

Ok. I will try to add some HTTP request mocks or perform real requests as the API I am using is intended for tests.